question16/main.js

```diff
@@ -41,35 +41,34 @@ async function init() {
 }
 init();
 
-function renderNewsUiElement(data) {
-  tabsGroup.appendChild(createTabs(data));
-  newsWrapper.appendChild(createContents(data));
+function renderNewsUiElement(newsData) {
+  tabsGroup.appendChild(renderTabElement(newsData));
+  newsWrapper.appendChild(renderContents(newsData));
 }
 
 //Render Tab Element
-const createTabs = (data) => {
+const renderTabElement = (newsData) => {
   const fragmentTablists = document.createDocumentFragment();
-  for (let i = 0; i < data.length; i++) {
+  for (let i = 0; i < newsData.length; i++) {
     const tabList = createElementWithClassName("li", "tabList");
-    tabList.textContent = data[i].category;
+    tabList.textContent = newsData[i].category;
     i === 0 && tabList.classList.add("is-active-tab");
```

Comment:
IMO
The task this time is
> どのカテゴリタブを初期表示時に選んでいるかはデータとして持っている

The specification was to have the initial display instruction data in the retrieved json. 0 is specified directly in the code, so please fix it if possible.

Reply:
I misunderstood the specifications...! Thank you for pointing this out! I changed the json data to have the initial display instruction data. It is like ``"initialDisplay": true`` and I changed it to have a class when it is true. Am I understanding you correctly?

Latest commit : https://github.com/csakyo/homework/pull/30/commits/e507fac706dd73eee0409c7c71f3f465a3a636ce